src/sentry/metrics/composite_experimental.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, **kwargs: Any):
             kwargs.pop("primary_backend", None), kwargs.pop("primary_backend_args", {})
         )
         self._allow_list = set(kwargs.pop("allow_list", set()))
+        self._deny_prefixes = tuple(kwargs.pop("deny_prefixes", []))
```

Comment:
Since `startswith` also accepts a tuple and we always have a static list defined in the settings, I would say we could use a tuple here. WDYT?

```suggestion
        self._deny_prefixes = tuple(kwargs.pop("deny_prefixes", tuple()))
```